### PR TITLE
Mobile View: Padding Adjust

### PR DESCRIPTION
**Pull request by lboogie04**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Adjust padding in the mobile view below the icons.
image

## Related Tickets & Documents
Does not close but addresses the Mobile View fix for Issue [19008](https://github.com/forem/forem/issues/19008)

- Related Issue #19008 
- Closes #

## QA Instructions, Screenshots, Recordings
Screenshot 2023-02-03 at 1 11 05 PM

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by rt4914 — CHANGES_REQUESTED** [review]
@lboogie04 

From the added image I was able to understand that following changes were expected in this PR:
- Top-padding for entire button: 8px
- Left-padding for entire button: 8px
- Right-padding for entire button: 8px
- Bottom-padding for entire button: 4px
- Gap/margin between the icon and count should be 4px
- Icon should be 32x32 px
- Text should be of 14 px

Based on these tasks I have found following issues:
1. The icon is not 32x32px instead its 24x24px
Screenshot 2023-02-06 at 10 44 26 PM
2. I think bottom padding for entire button is missing. Below are the calculations w.r.t height


- Currently, the entire button height is **58.09px**  (Image-1 attached below)
- Icon's height with padding is **35.98px** (Breakdown of this is: approx 24px icon height, 8px top padding, 4px bottom padding ~= 36px) (Image-2 attached below)
- Text height is **21px** (Image-3 attached below)
- Border top + bottom when icon is highlighted is ~= **1.11px** (Image-4 attached below)
- Total height (58.09px) = icon height + text height + borders = 35.98 + 21 + 1.11 = 58.09px
This calculations assures that the bottom margin of **4px** for entire button is missing.

| Name  | Image |
| ------------- | ------------- |
| Image 1 | Screenshot 2023-02-06 at 10 44 11 PM |
| Image 2 | Screenshot 2023-02-06 at 10 44 20 PM |
| Image 3 | Screenshot 2023-02-06 at 10 44 34 PM |
| Image 4 | Screenshot 2023-02-06 at 10 46 14 PM |

File: app/assets/stylesheets/views/article.scss [hunk 1]
```diff
@@ -254,6 +254,7 @@
 .reaction-drawer .crayons-reaction {
   border: 1px solid var(--card-bg);
   flex-direction: column;
+  padding: 8px 8px 4px 8px;
```
Comment (rt4914) [inline app/assets/stylesheets/views/article.scss:257]: This changes view for web too. 

### Result from this PR
Screenshot 2023-02-09 at 2 36 49 PM


### Result from Forem.team
Screenshot 2023-02-09 at 2 44 10 PM

File: app/helpers/crayons_helper.rb [hunk 2]
```diff
@@ -29,8 +29,8 @@ def crayons_icon_tag(name, native: false, **opts)
       opts.delete(:class),
     ].compact.join(" ")
 
-    opts[:width] ||= 24
-    opts[:height] ||= 24
+    opts[:width] ||= 32
+    opts[:height] ||= 32
```
Comment (rt4914) [inline app/helpers/crayons_helper.rb:33]: This is global setting and gets applied to all the crayons icon on the web, as a result we see following changes

### Result on this branch
Screenshot 2023-02-09 at 2 36 00 PM

### Result on dev.to
Screenshot 2023-02-09 at 2 36 11 PM

**Review by rt4914 — CHANGES_REQUESTED** [review]
Left some comments.

File: app/helpers/crayons_helper.rb (hunk 2 above)
Comment (rt4914) [reply app/helpers/crayons_helper.rb:33]: In `app/views/articles/_multireaction_button.html.erb` we make following change
Before: `<%= image_tag image_active_path, aria_hidden: true, height: 24, width: 24 %>`
After: `<%= image_tag image_active_path, aria_hidden: true, height: 32, width: 32 %>`

This should fix the icon size.

**Review by jaw6 — APPROVED** [review]
Looks okay to me, but would appreciate if @anujbhavsar96 can confirm

**Review by rt4914 — CHANGES_REQUESTED** [review]
I think it still needs some changes.

[Working Code](https://gist.github.com/rt4914/d784e829ef56c993c5d0ef4c7ad456b1)

### Mocks Screenshot
Screenshot 2023-02-10 at 10 27 08 PM

Screenshot 2023-02-10 at 10 50 13 PM


In this we can see that the approx size of the entire drawer is **312x93px**. I know we cannot make these exact size UI but still it can give us some idea.

### Output of this PR
|   |  |
| ------------- | ------------- |
| Screenshot 2023-02-10 at 10 23 31 PM  | Screenshot 2023-02-10 at 10 26 13 PM  |

This width and height for this PR is **394 x 105 px**


### Output of the attached code



|   |  |
| ------------- | ------------- |
| Screenshot 2023-02-10 at 10 24 37 PM | Screenshot 2023-02-10 at 10 25 07 PM |


This width and height for the attached PR is **312 x 95 px** [Working Code](https://gist.github.com/rt4914/d784e829ef56c993c5d0ef4c7ad456b1)

**Pull request merged**